### PR TITLE
New sauce recipe

### PR DESCRIPTION
**Pull request by knee-berts**
With love

### CODE REVIEW

File: Recipes/grandma-posidentos-sauce.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
...m only 1/4 Sicilian, my anonn came over to Brooklyn from Palermo during "the Great Arrival" and she was a constant reminder of the Italiano part of my heritage while living in her house during grammar school. This recipe started with Sicilian roots, but my annon was not your everyday Sicilian housewife. She was a master of organization, typing, and Latin (yes Latin), and found a career working in network television studios ghostwriting for execs. Often Grandma would come home regaling tails of collective bosses' inadequate grammar. So ya, Grandma is an OG feminist and I adore her for that. But the point is, she was not at home making sauce all day so the duty was shared with Gramps. Gramps was not Italian, he was a mix of German and Iroquis, and more importantly, and food engineer. The man knew flavors and iterated on his powerful wife's recipe to the point of perfection. This is my humble contribution to the community. Sicilian base layered with German ingenuity. This recipe defines comfort food for me and I hope the community can find some comfort in its preparation, and community-style serving. You make a giant pot and distribute it to your neighborhood - this is the way. Manja.
```
Comment (danpopnyc) [inline Recipes/grandma-posidentos-sauce.md:4]: "nonna"  typo

Comment (danpopnyc) [inline Recipes/grandma-posidentos-sauce.md:4]: Mangia vs "Manja"

File: Recipes/grandma-posidentos-sauce.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+3. When onion is soft (not carmelized) add garlic and saute until fragrant (about a minute).
+4. Add wine and burn off alcohol - I guess this takes a minute or two.
+5. Add tomatoes, basil, oregeno, sugar, salt, and stir together. Bring the sauce to a simmer for about half an hour. 
+6. Get an immersion blender, puree the sauce, add the sausages back in the pot and cook for no less than 5 hours while stirring every 30 mins of so. 
+7. TASTE IT, adjust, make it your own :) 
+
+### Meatballs
+1. Preheat oven to 400°F.
+2. In a large bowl combine water, bread crumbs, and eggs; stir to blend; let stand for 5 minutes. Add the meat and reamining ingredients.
+3. Shape meat into balls, place them on a lightley oiled sheet pan, and cook them for 5 minutes a side.
```
Comment (danpopnyc) [inline Recipes/grandma-posidentos-sauce.md:46]: "lightly"

**Review by danpopnyc — COMMENTED** [review]
couple o typos but this is a pretty amazing recipe dude!

**Comment by knee-berts** [comment]
Thanks Pop. I really appreciate your time here.

**Comment by danpopnyc** [comment]
i got you fam

**Review by danpopnyc — APPROVED** [review]
/lgtm

**Comment by knee-berts** [comment]
@markyjackson-taulia ^^

**Review by markyjackson-taulia — APPROVED** [review]
/lgtm

**Comment by knee-berts** [comment]
@thomcrowe - can I get an amen?

**Review by markyjackson-taulia — APPROVED** [review]

**Comment by knee-berts** [comment]
Bump

**Pull request merged**